src/datascience-ui/react-common/monacoEditor.tsx

```diff
@@ -632,6 +638,24 @@ export class MonacoEditor extends React.Component<IMonacoEditorProps, IMonacoEdi
         }
     }
 
+    private computeLineHeight(): number {
+        if (MonacoEditor.lineHeight <= 0 && this.state.editor) {
+            const editorDomNode = this.state.editor.getDomNode();
+            if (editorDomNode) {
+                const container = editorDomNode.getElementsByClassName('view-lines')[0] as HTMLElement;
+                const lineHeightPx =
+                    container.firstChild && (container.firstChild as HTMLElement).style.height
+                        ? (container.firstChild as HTMLElement).style.height
+                        : `${LINE_HEIGHT}px`;
+                MonacoEditor.lineHeight =
+                    lineHeightPx && lineHeightPx.endsWith('px')
+                        ? parseInt(lineHeightPx.substr(0, lineHeightPx.length - 2), 10)
```

Comment:
`parseInt` ignores trailing non numeric characters.
```suggestion
                        ? parseInt(lineHeightPx, 10)
```